langchain4j-cassandra/src/main/java/dev/langchain4j/store/embedding/cassandra/AstraDbEmbeddingStore.java

```diff
@@ -3,8 +3,13 @@
 import com.datastax.astra.sdk.AstraClient;
 import com.datastax.oss.driver.api.core.CqlSession;
 import com.dtsx.astra.sdk.cassio.MetadataVectorCassandraTable;
+import dev.langchain4j.data.embedding.Embedding;
```

Comment:
changes in this files seem to be by accident

Reply:
useless